Update main_data.py

Mantra0804 opened the pull request:
Added code to generate random date, old code was using the same date for all emails.
I have used randomtimestamp module to generate random date.

Mantra0804 committed:
Update main_data.py

Added code to genrate random date, old code was using same date for all emails

Mantra0804 committed:
Update main_data.py

Completed till 10th in part B

TarangRanpara reviewed (changes requested):
the changes you've done are of part B, and they have to kept in separate file named "main_test.py". otherwise everything's fine.

Mantra0804 committed:
Added main_test.py

remove code of partB from main_data.py
added main_test.py for partB

Mantra0804 committed:
Update main_data.py

Made few changes in ramdom date genrater.
Removed random.seed as it genrating same random number

Mantra0804 committed:
Update main_data.py

Mantra0804 commented:
I have added main_test.py for partB
Removed random.seed as it was generating same random number

TarangRanpara reviewed (approved):
i am merging this changes
